magento/pwa-studio
# [PWA-834] Localize My Account & Signed In Sidebar

## Description

Translate strings within the My Account and Signed in Sidebar

## Related Issue





https://jira.corp.magento.com/browse/PWA-834

## Acceptance 



### Verification Stakeholders
@davemacaulay 
@dpatil-magento 
@jimbo 

### Specification



### Verification Steps


1. Go to sections: My Account, and Signed In sidebar
2. Verify strings are translated where applicable.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
